$MirOS: ports/emulators/qemu/stable/patches/patch-target-arm_cpu_h,v 1.1 2008/05/07 15:46:12 tg Exp $
--- target-arm/cpu.h.orig	Wed Apr 27 20:51:43 2005
+++ target-arm/cpu.h	Tue May  3 18:04:53 2005
@@ -104,7 +104,6 @@ void cpu_arm_close(CPUARMState *s);
    signal handlers to inform the virtual CPU of exceptions. non zero
    is returned if the signal was handled by the virtual CPU.  */
-struct siginfo;
-int cpu_arm_signal_handler(int host_signum, struct siginfo *info, 
+int cpu_arm_signal_handler(int host_signum, siginfo_t *info, 
                            void *puc);
 
 #define TARGET_PAGE_BITS 12
